src/mono/sample/wasm/browser-es6/Wasm.Browser.ES6.Sample.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration>Debug</Configuration>
     <WasmCopyAppZipToHelixTestDir Condition="'$(ArchiveTests)' == 'true'">true</WasmCopyAppZipToHelixTestDir>
-    <WasmMainJSPath>main.js</WasmMainJSPath>
+    <WasmMainJSPath>main.mjs</WasmMainJSPath>
```

Comment:
I see the `.cjs` extension was dropped. Isn't it a similar case here?

Reply:
Will do it here too.